recipes/agent_linux.rb

```diff
@@ -31,7 +31,7 @@
 server_search_query     = node['gocd']['agent']['server_search_query']
 
 package "go-agent" do
-  version node['gocd']['version']
+  version node['gocd']['version'].split('-')[0]
```

Comment:
I don't think this is a good idea. It's not obvious if people want to skip a certain build for some reason, or if the version string is missing the build #.